tests/test_ingestion_manifest.py

```diff
@@ -0,0 +1,306 @@
+# pylint: disable=missing-docstring
+# mypy: disable-error-code="no-untyped-def"
+
+import json
+from datetime import datetime
+from pathlib import Path
+
+import pytest
+
+from src.blueprints.common_models import GroupACL, UserACL
+from src.blueprints.datafile import RawDatafile
+from src.blueprints.dataset import RawDataset
+from src.blueprints.experiment import RawExperiment
+from src.blueprints.project import RawProject
+from src.extraction.manifest import IngestionManifest
+from src.mytardis_client.enumerators import DataClassification
+from src.utils.filesystem.filesystem_nodes import DirectoryNode
+
+# pylint: disable=line-too-long
+# NOTE: pyfakefs's FakeFilesystem does not currently work for these tests, because
+#       of an interaction with pathlib. This should apparently be fixed with changes
+#       to pathlib in Python 3.11.
+#       https://pytest-pyfakefs.readthedocs.io/en/latest/troubleshooting.html#pathlib-path-objects-created-outside-of-tests
+
+
+@pytest.fixture(name="ingestion_manifest")
```

Comment:
These are very complete and probably less out of date than the raw_dataclass classes in fixtures_dataclasses.py, so it might be worth making a new ticket ticket to update the dataclass fixtures by separating one of each class in this very big fixture out into individual RawDataclass fixtures.

Reply:
That's a good point, the fixture is definitely large. I will make a ticket for splitting it up